fluent-plugin-protobuf/lib/fluent/plugin/parse_protobuf.rb

```diff
@@ -13,16 +13,14 @@ class ProtobufParse < Fluent::Plugin::Parser
       def parse(text)
         begin
           inflated = Snappy.inflate(text)
```

Comment:
open question - all Prometheus write API streams are in same compressing and encoding settings?

Reply:
I believe so. It seems that other Prometheus remote write integrations assume the same for compressing and encoding